content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
 weight: 300
 toc: true
 ---
+
+### Settings Registry
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the content processing rules that **Asset Processor** uses for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.

```

Comment:
"This file is well commented, and full of examples you can use for *to update/to customize/to set* specific configurations on your project's asset pipeline."?